src/background.ts

```diff
@@ -52,17 +53,24 @@ function createWindow () {
     }
   })
 
-  mainWindowState.manage(win)
+  win.webContents.session.setProxy({
+    proxyRules: `socks5://127.0.0.1:${TOR_SOCKS_PORT}`,
+    proxyBypassRules: '<local>, 192.168.1.1/16, fefe:13::abc/33'
+  }).then(() => {
+    console.log('Socks proxy set')
+
+    mainWindowState.manage(win!)
```

Comment:
window state needs to be outside this callback.
It needs to be there before window.show() otherwise it won't use the saved window state because its already rendered. I suggest putting it back where it was right after `win = new BrowserWindow({`